### PR TITLE
Document Docker memory requirements in Getting Started

### LINKED ISSUE #2811: Document memory requirements to Getting Started guide
This has been a recurring issue. Developers will need to allocate ~4GB of memory to Docker in order to run and initialize all the Warehouse services.

### PR DESCRIPTION
**Pull request by pradyunsg**
Fixes #2811

### CODE REVIEW

File: docs/development/getting-started.rst [hunk 1]
```diff
@@ -98,16 +98,23 @@ mounted inside of the docker container at ``/opt/warehouse/src/``.
...
-one terminal run the command:
+Then, in one terminal run the command:
 
 .. code-block:: console
 
     make serve
 
+After building the Docker container, you'll need to create a Postgres database
+and run all of the data migrations.
+
```
Comment (di) [inline docs/development/getting-started.rst]: This line is sort of repeating what comes after it, could you revise this a little more? Or possibly just eliminate it.

Comment (pradyunsg) [reply docs/development/getting-started.rst]: How does it read now? :)

**Comment by di** [comment]
Thanks @pradyunsg! Send ernest@python.org an email with a link to this PR and he'll get some stickers out to you!

**Comment by pradyunsg** [comment]
Donezo. =)

**Pull request merged**